test/Microsoft.ApplicationInsights.AspNet.Tests/FakeTelemetryChannel.cs

```diff
@@ -6,6 +6,8 @@
     // TODO: Remove FakeTelemetryChannel when we can use a dynamic test isolation framework, like NSubstitute or Moq
```

Comment:
@abaranch, as far as I can tell our tests are not running on dnxcore50 even now.
@SergKanz, Moq has the same problem - it supports net40, but not dnxcore50.